src/test/memfs.test.ts

```diff
@@ -18,6 +18,14 @@ describe('memfs.ts', () => {
                 sinon.assert.calledOnce(listener)
                 sinon.assert.calledWithExactly(listener, 'file:///foo/bar.txt', undefined)
             })
+            it('should add a URI with `%` character and emit an event', () => {
```

Comment:
This test would have been failed before this PR changes.
This is exactly the case #422 referres to.